content/en/docs/_index.md

```diff
@@ -6,11 +6,11 @@ weight: 1
 
 ## About Falco
 
-Falco is a behavioral activity monitor designed to detect anomalous activity in your applications. Using powerful [system call capture](https://sysdig.com/blog/fascinating-world-linux-system-calls/) technology originally built by Sysdig. Falco lets you continuously monitor and detect container, application, host, and network activity, all in one place, from one source of data, with one set of [rules](rules).
+**Falco** is a runtime security tool designed for [OCI]-compliant containers. It uses powerful [system call capture][sys_call_capture] technology, originally built by [Sysdig], to monitor the [behavior](#behaviors) of applications—including container, application, host, and network activity—all in one place and from one source of data. You can specify which behavior you consider anomalous using Falco [rules].
```

Comment:
Falco is a runtime security tool designed for [OCI]-compliant containers. It uses the powerful [system call capture][sys_call_capture] technology, originally built by [Sysdig], to monitor the [behavior](#behaviors) of applications—including container, application, host, and network activity—all in one place and from one source of data. You can specify which behavior you consider anomalous using Falco [rules].